protocols/secio/Cargo.toml

```diff
@@ -20,7 +20,7 @@ libp2p-core = { version = "0.16.0", path = "../../core" }
 log = "0.4.6"
 prost = "0.6.1"
 pin-project = "0.4.6"
-quicksink = "0.1"
+quicksink = { git = "https://github.com/paritytech/quicksink", branch = "develop" }
```

Comment:
If this PR is approved, the new quicksink version will be published to crates.io and this entry will become `quicksink = "0.2"`.